tests/auth/mfa_modules/test_insecure_example.py

```diff
@@ -121,7 +121,7 @@ async def test_login(hass: HomeAssistant) -> None:
     )
     assert result["type"] == data_entry_flow.FlowResultType.FORM
     assert result["step_id"] == "mfa"
-    assert result["data_schema"].schema.get("pin") == str
+    assert isinstance(result["data_schema"].schema.get("pin"), str)
```

Comment:
The transform here is probably wrong. Instead of isinstance it should've been `result[...]... is str`. Same for other case below.

Reply:
Yes, that's my fault.